### PR TITLE
store dashboard with rendering of managing listings and orders

### PR DESCRIPTION
**Pull request by iasssy**

### CODE REVIEW

File: src/pages/StoreDashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
```
Comment (hdcola) [inline src/pages/StoreDashboard.tsx]: you can only import useState

File: src/components/Dashboard/SideNav.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
```
Comment (hdcola) [inline src/components/Dashboard/SideNav.tsx:1]: you can only import useState

**Review by hdcola — CHANGES_REQUESTED** [review]
Try not to import *, especially when you actually only use one

File: src/pages/StoreDashboard.tsx (hunk 1 above)
Comment (iasssy) [reply src/pages/StoreDashboard.tsx]: fixed, thank you!

File: src/components/Dashboard/SideNav.tsx (hunk 2 above)
Comment (iasssy) [reply src/components/Dashboard/SideNav.tsx:1]: ok!

**Review by hdcola — APPROVED** [review]

**Pull request merged**